include/build_config/SDL_revision.h.cmake

```diff
@@ -31,9 +31,21 @@
 #cmakedefine SDL_VENDOR_INFO "@SDL_VENDOR_INFO@"
 
 #ifdef SDL_VENDOR_INFO
-#define SDL_REVISION "@SDL_REVISION@ (" SDL_VENDOR_INFO ")"
+#cmakedefine SDL_REVISION "@SDL_REVISION@ (" SDL_VENDOR_INFO ")"
 #else
-#define SDL_REVISION "@SDL_REVISION@"
+#cmakedefine SDL_REVISION "@SDL_REVISION@"
+#endif
+
+#ifndef SDL_REVISION
+#ifdef SDL_VENDOR_INFO
+#define SDL_REVISION SDL_VENDOR_INFO
+#else
+#define SDL_REVISION ""
+#endif
+#endif
+
+#ifndef SDL_VENDOR_INFO
+#define SDL_VENDOR_INFO ""
```

Comment:
This has changed in current SDL3 main.
The CMake script always defines a `SDL_REVISION` string, and optionally adds a vendor info string.

Please verify, but I think the current SDL_revision.h.cmake can remain as is.